ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -55,3 +57,31 @@ def send_metric(self, metric_type, metric_name, metric_value, tags):
             getattr(self, metric_type)(metric_name, metric_value, tags=tags)
         else:
             self.log.warning("Unknown metric type `%s` for metric `%s`", metric_type, metric_name)
+
+    def _collect_metadata(self):
+        raw_version = self._get_version()
+        if not raw_version:
+            self.log.warning("Version not found in stdout")
```

Comment:
```suggestion
            self.log.debug("Version not found in stdout")
```

in previous implementation, warning level logs have been known to be too noisy